fitlins/interfaces/bids.py

```diff
@@ -159,6 +159,8 @@ class LoadBIDSModel(SimpleInterface):
     output_spec = LoadBIDSModelOutputSpec
 
     def _run_interface(self, runtime):
+        import bids
+        bids.config.set_options(loop_preproc=True)
```

Comment:
@adelavega Does this look like the right place to do this? My understanding is the time to do it is before `bids.Analysis().setup()` is called. Are there any other cases where it's important to ensure this is set?

Reply:
That seems right to me. The actual function that uses this is `load_variables` so as long as its done prior to that, it should work.